scripts/cc-cli/src/utils/bond.ts

```diff
@@ -0,0 +1,37 @@
+import { ApiPromise } from "creditcoin-js";
+import { initKeyringPair } from "./account";
+
+export async function bond(
+  stashSeed: string,
+  controllerAddress: string,
+  amount: number,
+  rewardDestination: "Staked" | "Stash" | "Controller",
+  api: ApiPromise
+) {
+  const bondTx = api.tx.staking.bond(
+    controllerAddress,
+    `${amount}000000000000000000`, // Add 18 zeros to convert to micro units TODO: Improve amount handling
```

Comment:
hmmm. What of I type `123` followed by a random number of zeroes (possibly 18) as input ? 

I think we should be handling numbers as numbers instead of as strings. 

Also some basic error checking may be good here. What if I specify a negative amount (could already be handled on the BE side but still, not a reason for not checking it) ?